examples/remote-config/remote_config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "command_repository_base_url": "https://raw.githubusercontent.com/criteo/command-launcher/update-command/test/remote-repo",
+  "command_repository_base_url": "https://raw.githubusercontent.com/criteo/command-launcher/distributed-pkg/test/remote-repo",
```

Comment:
Once this is merged, you will need another review to change the branch you are targeting for tests before deleting this branch, right ?

Reply:
Yes